taikun/resource_taikun_project.go

```diff
@@ -1208,3 +1214,63 @@ func resourceTaikunProjectServerKubernetesLabels(data map[string]interface{}) []
 	}
 	return labelsToAdd
 }
+
+func resourceTaikunProjectValidateKubernetesProfileLB(data *schema.ResourceData, apiClient *apiClient) error {
+	if kubernetesProfileIDData, kubernetesProfileIsSet := data.GetOk("kubernetes_profile_id"); kubernetesProfileIsSet {
+		kubernetesProfileID, _ := atoi32(kubernetesProfileIDData.(string))
+		lbSolution, err := resourceTaikunProjectGetKubernetesLBSolution(kubernetesProfileID, apiClient)
+		if err != nil {
+			return err
+		}
+		if lbSolution == loadBalancerNone {
+			return nil
+		}
+
+		cloudCredentialID, _ := atoi32(data.Get("cloud_credential_id").(string))
+		cloudType, err := resourceTaikunProjectGetCloudType(cloudCredentialID, apiClient)
+		if err != nil {
+			return err
+		}
+		if cloudType != cloudTypeOpenStack {
+			return fmt.Errorf("If %s load balancer is enabled, cloud type should be OpenStack; is %s", lbSolution, cloudType)
+		}
```

Comment:
In the UI and in API, It's not an error to have Taikun or Octavia enabled in a non OpenStack project. It just won't work